test/SubdomainWithDCLControllerV2.spec.js

```diff
@@ -176,6 +178,7 @@ describe('DCL Names V2 with DCLControllerV2', function () {
     dclControllerContract = await DCLControllerV2.new(
       manaContract.address,
       dclRegistrarContract.address,
+      feeCollector,
       creationParams
```

Comment:
is it this param still required in the constructor?

Reply:
my bad, I meant: `creationParams`

Reply:
It is needed because it tells which signer is deploying the contract (the deployer determines the owner)

```
creationParams = {
      ...fromDeployer,
```